FND/DatesAndTimes/FinancialDates.rdf

```diff
@@ -608,20 +640,20 @@ The recurrence start date can also be relative to the starting Date of the overa
 		<rdfs:subPropertyOf rdf:resource="&lcc-lr;has"/>
 		<rdfs:label>has calendar period</rdfs:label>
 		<rdfs:range rdf:resource="&fibo-fnd-dt-fd;CalendarPeriod"/>
-		<skos:definition>identifies a calendar period used in computing a CalendarSpecifiedDate, such as a calendar week, calendar month, calendar quarter, or calendar year</skos:definition>
+		<skos:definition>identifies a period of time used in computing a calendar-specified date, such as a calendar week, calendar month, calendar quarter, or calendar year</skos:definition>
 	</owl:ObjectProperty>
 	
 	<owl:DatatypeProperty rdf:about="&fibo-fnd-dt-fd;hasCount">
 		<rdfs:label>has count</rdfs:label>
 		<rdfs:range rdf:resource="&xsd;positiveInteger"/>
-		<skos:definition>indicates the number of things in a collection, such as the number of entries in a RegularSchedule</skos:definition>
+		<skos:definition>indicates the number of things in a collection, such as the number of entries in a regular schedule</skos:definition>
 	</owl:DatatypeProperty>
 	
 	<owl:ObjectProperty rdf:about="&fibo-fnd-dt-fd;hasDate">
 		<rdfs:subPropertyOf rdf:resource="&lcc-lr;has"/>
```

Comment:
not "has" here or in the subsequent properties

Reply:
We said that property names should be verbs.  In this case since it explicitly says how many, I'll replace 'indicates' with 'specifies'.  But I don't understand what is wrong with indicates.